deploy/template.yaml

```diff
@@ -441,6 +441,7 @@ Resources:
               ACCOUNT_PHONE_OTP: "/perfTest/account/fixedPhoneOTP"
               ACCOUNT_RP_STUB: "/perfTest/account/authentication/rpStub"
               ACCOUNT_SIGNIN_URL: "/perfTest/account/accounts/signInUrl"
+              ACCOUNT_STAGING_URL: "/perfTest/account/authentication/baseUrl"
```

Comment:
Is this a different value to `ACCOUNT_SIGNIN_URL`?

Reply:
yes, `ACCOUNT_SIGNIN_URL` is for build environment (`https://signin.build.account.gov.uk`)